localization/languages/nl.json

```diff
@@ -0,0 +1,251 @@
+        "sslTimeErrorMessage": "Min kon niet veilig verbinden met deze website. Kijk na of de klok op uw computer correct is ingesteld",
+        "addressInvalidTitle": "Dit adres is ongeldig.",
+        "genericError": "Er ging iets mis",
+        /* pages/phishing/index.html */
+        "phishingErrorTitle": "Deze website kan schadelijk zijn.",
+        "phishingErrorMessage": "Deze website probeert mogelijk persoonlijke informatie te stelen zoals wachtwoorden of gegevens over uw financiële transacties.",
+        "phishingErrorVisitAnyway": "Website toch bezoeken",
+        "phishingErrorLeave": "Website verlaten",
+        /* multiple instances alert */
+        "multipleInstancesErrorMessage": "Er ging iets mis. Sluit a.u.b. alle andere open vensters van Min af en herstart Min.",
+        /* pages/sessionRestoreError/index.html */
+        "sessionRestoreErrorTitle": "Er ging iets mis",
+        "sessionRestoreErrorExplanation": "Uw opgeslagen tabbladen konden niet correct hersteld worden",
+        "sessionRestoreErrorBackupInfo": "Min heeft een backup opgeslagen van uw gegevens op deze locatie: %l.", //%l will be replaced with a path to a file
+        "sessionRestoreErrorLinkInfo": {
+            "unsafeHTML": "Open a.u.b. een nieuwe foutmelding op <a href=\"https://github.com/minbrowser/min\" target=\"_blank\">here</a> als deze fout zich blijft herhalen."
+        },
+        /* pages/settings/index.html */
+        "settingsPreferencesHeading": "Voorkeuren",
+        "settingsRestartRequired": "Herstart Min a.u.b. om deze wijzigingen toe te passen.",
+        "settingsPrivacyHeading": "Inhoud blokkeren",
+        "settingsContentBlockingLevel0": "Alle advertenties en trackers toestaan",
+        "settingsContentBlockingLevel1": "Advertenties en trackers van derden blokkeren",
+        "settingsContentBlockingLevel2": "Alle advertenties en trackers blokkeren",
+        "settingsContentBlockingExceptions": "Advertenties worden toegestaan op volgende websites:",
+        "settingsBlockScriptsToggle": "Blokkeer scripts",
+        "settingsBlockImagesToggle": "Blokkeer afbeeldingen",
+        "settingsEasyListCredit": {
+            "unsafeHTML": "Gebaseerd op EasyList en EasyPrivacy, gecreëerd door <a target=\"_blank\" href=\"https://easylist.to/\">de EasyList auteurs</a> (<a target=\"_blank\" href=\"https://creativecommons.org/licenses/by-sa/3.0/legalcode\">view license</a>)."
+        },
+        "settingsAppearanceHeading": "Uitzicht",
+        "settingsEnableDarkMode": "Donkere modus inschakelen:",
+        "settingsDarkModeNever": "Nooit",
+        "settingsDarkModeNight": "'s Nachts",
+        "settingsDarkModeAlways": "Altijd",
+        "settingsSiteThemeToggle": "Kleurthema van site inschakelen",
+        "settingsAdditionalFeaturesHeading": "Aanvullende opties",
+        "settingsUserscriptsToggle": "Gebruikerscripts inschakelen",
+        "settingsSeparateTitlebarToggle": "Toon de titelbalk",
```

Comment:
This is a pretty minor issue, and I'm reviewing this by using Google Translate and looking at a dictionary (so I could be totally wrong), but if I understand correctly this is "Show the titlebar" - it might make more sense to include "separate" here? There's always a titlebar shown regardless of this setting; the setting just controls whether the titlebar is combined in the same line as the other controls or is on a separate line ([screenshots](https://github.com/minbrowser/min/pull/977#discussion_r414918201)).

Reply:
Correct, I'll change that to 'Toon aparte titelbalk'